### DISCUSSION TITLE
Access artist page from song

### DISCUSSION THREAD

**Discussion by Billuc** [KRTirtho/spotube#1320] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I like going to artist pages to see their most popular songs or see other albums. However, I haven't found a nice way of directly going there. My workaround for now is to play a song of this artist and tap on the artist name on the play screen.

### Suggested solution

Maybe a 'Go to artist page' button in a song context menu, similar to 'Go to album'. Since there can be multiple artists, a strategy has to be chosen, like ask which artist page to go to (like Spotify) or go to the main artist page (like Youtube Music). 

### Useful resources

I am using Spotube on Android (Samsung Galaxy S10 SE).

### Additional information

I may be able to work on this in my free time (hopefully when I have some), if you give me pointers ;)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
It's actually really straightforward. You can directly click/tap the track's artist's names

![Image](https://github.com/KRTirtho/spotube/assets/61944859/ff29d9b3-ad3b-4249-b429-ff727e352af3)
